app/assets/javascripts/lib/ui_helper.js

```diff
@@ -0,0 +1,69 @@
+Skull.UI_helper = {
```

Comment:
The whole indentation here is weird. We use the same defaults for ruby and JS: 2 spaces